src/common/utility.js

```diff
@@ -244,3 +244,15 @@ export const redirectOpenLiveChatBox = (is_redirect) => {
         navigate(live_chat_redirection_link, { replace: true })
     }
 }
+
+export const addScript = (settings) => {
+    const script = document.createElement('script')
+    const { async, text, src, id } = settings
+
+    if (async) script.async = async
+    if (text) script.text = text
+    if (src) script.src = src
+    if (id) script.id = id
```

Comment:
Can we apply the settings in one step like this?

```suggestion
    let script = document.createElement('script');
    
    script = {
        ...script,
        ...settings,
    }
```

Reply:
Yes @mahdiaryayi-fs  that would be cleaner

Reply:
Hi @mahdiaryayi-fs  found out that the script is a`DOM` element and not an object we can just merge by spreading. I did another work around instead.